instance/models/instance.py

```diff
@@ -89,6 +92,18 @@ def save(self, *args, **kwargs):
             'instance_id': self.pk,
         })
 
+    @classmethod
+    def can_manage(cls, user):
+        """
+        Returns true if the user is allowed to manage all instances.
+
+        User must:
+        * be a staff member, or
+        * have the "manage_all" permission
+        """
+        permission = '{}.{}'.format(cls._meta.app_label, "manage_all")
+        return user.is_staff or user.has_perm(permission)
```

Comment:
@bradenmacdonald @smarnach You're correct, the model access rights work fine without checking for `is_staff`.

However, since I'm using this `can_manage` method for the [ApiInstanceManagerPermission](https://github.com/open-craft/opencraft/pull/104/files#diff-ff05414fb9cbd08c954e674eef451fc5R32) and the [opencraft IndexView](https://github.com/open-craft/opencraft/pull/104/files#diff-8d4846bd50606dc2973def39917f5f68R33) redirect as well, then it helps me to keep it in one place.